superpixel_classification/SuperpixelClassification/SuperpixelClassificationBase.py

```diff
@@ -795,12 +795,14 @@ def main(self, args):
                 gc, args.images, args.annotationName, args.features, args.patchSize,
                 args.numWorkers, prog)
 
+            batchSize = args.batchSize
+            batchSize = -1  # Always optimize the batch size
```

Comment:
Is there a reason this is here, or is it leftover from testing?  And, I think we should change the default value in the xml.

Reply:
I put it there so that it would be easy for reviewers to test.  I'll pull it out before we merge this.

Reply:
> And, I think we should change the default value in the xml.

Will do.

Reply:
I've removed the setting of batchsize to -1.  It's all up to the caller now, as it should be, though the default is now -1 in the `xml` file.